x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/action/DataStreamsStatsAction.java

```diff
@@ -37,7 +37,7 @@ public DataStreamsStatsAction() {
         public Request() {
             // this doesn't really matter since data stream name resolution isn't affected by IndicesOptions and
             // a data stream's backing indices are retrieved from its metadata
-            super(null, IndicesOptions.fromOptions(false, true, true, true, false, false, true, false));
+            super(null, IndicesOptions.fromOptions(false, true, true, true, true, false, true, false));
```

Comment:
These options are the same for the GetDataStreams API, do those need to be updated as well?

Reply:
I don't think so because backing indices aren't being resolved in that API.

Reply:
That's a good question, I don't think so (or else I would have expected us to find it much earlier) since those are retrieved differently?